pkg/webhook/validator.go

```diff
@@ -254,13 +255,31 @@ func (v *Validator) validatePodSpec(ctx context.Context, namespace string, ps *c
 						}
 					}
 				}
-				logging.FromContext(ctx).Errorf("policies: for %v", policies)
 			}
 
 			if passedPolicyChecks {
 				logging.FromContext(ctx).Debugf("Found at least one matching policy and it was validated for %s", ref.Name())
 				continue
 			}
+			// There were no matching policies found against this image.
+			// Check what the configuration is and act accordingly.
+			pcConfig := policycontrollerconfig.FromContext(ctx)
```

Comment:
nit: I'd move this code to its own function, this `validatePodSpec` is getting big.

Reply:
done